### PR TITLE
Instream adapter listener for suppressed autostart

### PR DESCRIPTION
**Pull request by boyntoni**
### This PR will...

Add a listener on the instream adapter to handle suppressed autostart attempt on ads. 

### Why is this Pull Request needed?

Freewheel had not been reflecting a paused state after an ad paused from a suppressed autostart. 

### Are there any points in the code the reviewer needs to double check?

I am currently using the MEDIA_PLAY_ATTEMPT_FAILED, but believe a new event would be best to capture this occurrance. 

### Are there any Pull Requests open in other repos which need to be merged with this?

Yes, 

#### Addresses Issue(s):

ADS-891

### CODE REVIEW

**Comment by johnBartos** [comment]
test this please

**Comment by robwalch** [comment]
@boyntoni @egreaves Let's try to get this in v8.1.6. It would be good to get in and verify before the autoplay test changes. Even with the test preventing autoplay, play attempts could fail for a few reasons and this insures the player state is correctly displayed if that happens.

Edge case examples: a second play request, pause or load is called on the video tag while buffering...

File: src/js/controller/instream-adapter.js [hunk 1]
```diff
@@ -126,6 +127,10 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
         return this;
     };
 
+    function triggerPlayRejected() { 
+        _model.set('playRejected', true);
```
Comment (robwalch) [inline src/js/controller/instream-adapter.js]: Did you try setting this on the instream model?

Comment (boyntoni) [reply src/js/controller/instream-adapter.js]: @robwalch  The reason I triggered it on the model was to hit the listener on the PlayDisplayIcon, which reacts to changes in state -- i.e. the goal was to get the play display icon to show.

Comment (robwalch) [reply src/js/controller/instream-adapter.js]: That uses the view-model which should get the change from instream's model when it's active. Try it please.

**Comment by johnBartos** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  🛠 There are modified src files, but no test changes. Add tests if you're able to.
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  🔎 Assign some reviewers or assignees.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>
